src/images/pages/trade/High_ICON.svg

```diff
@@ -0,0 +1 @@
+<svg width="48" height="48" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg"><title>High ICON</title><g fill="none" fill-rule="evenodd"><path fill="#2AAAE2" d="M0 48h48V0H0z"/><path fill="#FFF" d="M0 33h48v-2H0z"/><path stroke="#FFF" stroke-width="4" d="M5 10l11.477 18L34 10"/><path fill="#FFF" d="M36 8h-8l-3.5 3.5 5.126.094 2.824 2.854.05 5.052L36 16z"/><path d="M41 29V7m-3 19l3 3 3-3m-6-16l3-3 3 3" stroke="#FFF" stroke-width="2"/></g></svg>
```

Comment:
can we change all the image file names? without any caps and standardize `-` or `_`, at least between these 3 image files

Reply:
sure.

Reply:
I think you changed the reference but not the file name for high_icon and low_icon :)

Reply:
That's git cache :/

Reply:
https://stackoverflow.com/a/17688308/4425240